src/components/Filter.tsx

```diff
@@ -28,12 +26,54 @@ export const Filter = (props: IProps) => {
   return (
     <section className="filter">
       <div>
-        <button onClick={showAll}>All</button>
-        <button onClick={() => filter('ui')}>UI</button>
-        <button onClick={() => filter('ux')}>UX</button>
-        <button onClick={() => filter('enhancement')}>Enhancement</button>
-        <button onClick={() => filter('bug')}>Bug</button>
-        <button onClick={() => filter('feature')}>Feature</button>
+        <input
+          type="radio"
+          id="all"
+          name="filter"
+          value="all"
+          onClick={showAll}
+        ></input>
```

Comment:
This looks much more semantic! Nicely done. A couple of recommendations:
- We can see that these are repeated quite a few times. Maybe you could switch this to use a `.map`?
- With JSX you can have self-closing tags. So `<input ...></input>` can become `<input ... />`
- Should there be a `defaultChecked`? Otherwise, nothing is selected by default
- Is there a need to wrap `div` around all of these inputs?

Reply:
That was definitely what I was thinking about last night after I submitted the PR for this feature. Will make those changes!

Reply:
Just pushed up some changes for the refactor